tests/h2-tests/tests/flow_control.rs

```diff
@@ -1891,9 +1891,7 @@ async fn window_size_decremented_past_zero() {
         let builder = server::Builder::new();
         let mut srv = builder.handshake::<_, Bytes>(io).await.expect("handshake");
 
-        // just keep it open
-        let res = poll_fn(move |cx| srv.poll_closed(cx)).await;
```

Comment:
Is this bug covered by this test? If not, is it possible to cover this bug in a test?

Reply:
Yes, it returned an error prior to my changes, now it doesn’t and it checks that it doesn’t.